sdk/src/wallet/bindings/java/examples/src/RequestFundsFromFaucet.java

```diff
@@ -22,6 +22,8 @@ public static void main(String[] args) throws WalletException, InterruptedExcept
                 .withStoragePath(Env.STORAGE_PATH)
         );
 
+        wallet.setStrongholdPassword(Env.STRONGHOLD_PASSWORD);
```

Comment:
Where's this change coming from?

Reply:
removed

Reply:
There were a few instances I noted. Are any of them supposed to be there?

Reply:
not in java for sure